tools/wptrunner/wptrunner/browsers/safari.py

```diff
@@ -193,15 +193,22 @@ def stop(self, force=False):
         if self.kill_safari:
             self.logger.debug("Going to stop Safari")
             for proc in psutil.process_iter(attrs=["exe"]):
-                if (proc.info["exe"] is not None and
-                    os.path.samefile(proc.info["exe"], self.safari_path)):
-                    self.logger.debug("Stopping Safari %s" % proc.pid)
+                if proc.info["exe"] is None:
+                    continue
+
+                try:
+                    if not os.path.samefile(proc.info["exe"], self.safari_path):
+                        continue
+                except OSError:
+                    continue
+
+                self.logger.debug("Stopping Safari %s" % proc.pid)
```

Comment:
Should this either be outside the `for` loop, or break after we kill Safari?

Reply:
This isn't a change from previously, and I'd rather avoid increasing scope of this. I think there's reasonable bugs to file here (e.g., also terminating other related processes), but I don't think this is worthwhile changing right now.